.github/workflows/main.yml

```diff
@@ -8,6 +8,6 @@ on:
 
 jobs:
   main:
-    uses: asottile/workflows/.github/workflows/tox.yml@v1.7.0
+    uses: asottile/workflows/.github/workflows/tox.yml@v1.8.1
```

Comment:
This cherry-pick is intentional - CI won't run with version v1.7.0 at all because ["it uses a deprecated version of `actions/upload-artifact: v3`"](https://github.com/asottile/all-repos/actions/runs/13100000673/job/36547066682)